pyvista/plotting/mapper.py

```diff
@@ -25,17 +25,20 @@ class _BaseMapper(_vtk.vtkAbstractMapper):
 
     _theme = None
 
-    def __init__(self, theme, **kwargs):
+    def __init__(self, theme=None, **kwargs):
         self._theme = theme
         self.lookup_table = LookupTable()
 
-    def copy(self):
-        """Create a deep of this mapper.
+    def copy(self, deep=True) -> '_BaseMapper':
+        """Create a copy of this mapper.
+
+        Parameters
+        ----------
+        deep : bool, optional
+            Whether to return a deep or a shallow copy.
```

Comment:
The deep argument is not used. Will it be added in the future?

Reply:
Good catch @tkoyama010. I've removed this in 48f0f1e5be8bd077481b6a56ea077002bf6599de.